<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileTransfer</class>
 <widget class="QMainWindow" name="FileTransfer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文件传输</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="targetAddressEdit">
      <property name="placeholderText">
       <string>请输入目标地址</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="portNumberLabel">
      <property name="text">
       <string>端口号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="portNumberEdit">
      <property name="placeholderText">
       <string>请输入端口号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="filePathLabel">
      <property name="text">
       <string>文件路径：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="selectFileButton">
        <property name="text">
         <string>选择文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendFileButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>发送文件</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QLabel" name="connectionStatusLabel">
      <property name="text">
       <string>未连接</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="connectButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>连接</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>

